go.mod

```diff
@@ -20,7 +20,7 @@ require (
 	k8s.io/component-base v0.27.1
 	k8s.io/controller-manager v0.27.1
 	k8s.io/klog/v2 v2.90.1
-	k8s.io/kubernetes v1.27.1
```

Comment:
@atiratree  Is this `v1` typo or intended?

Reply:
@ardaguclu seems this intended, can you please update once more? 

https://pkg.go.dev/k8s.io/kubernetes?tab=versions